src/sqlancer/stonedb/ast/StoneDBConstant.java

```diff
@@ -0,0 +1,203 @@
+package sqlancer.stonedb.ast;
+
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+
+import sqlancer.common.ast.newast.Node;
+
+public class StoneDBConstant implements Node<StoneDBExpression> {
+
+    private StoneDBConstant() {
+    }
+
+    public static class StoneDBNullConstant extends StoneDBConstant {
+        @Override
+        public String toString() {
+            return "NULL";
+        }
+    }
+
+    public static Node<StoneDBExpression> createNullConstant() {
+        return new StoneDBNullConstant();
+    }
+
+    public static class StoneDBIntConstant extends StoneDBConstant {
+        private final Integer value;
+
+        public StoneDBIntConstant(int value) {
+            this.value = value;
+        }
+
+        @Override
+        public String toString() {
+            if (value.equals(Integer.MIN_VALUE)) {
+                return "INT_NULL";
+            }
+            return String.valueOf(value);
+        }
+    }
+
+    public static Node<StoneDBExpression> createIntConstant(int val) {
+        return new StoneDBIntConstant(val);
+    }
+
+    public static class StoneDBBigIntConstant extends StoneDBConstant {
+        private final Long value;
+
+        public StoneDBBigIntConstant(long value) {
+            this.value = value;
+        }
+
+        @Override
+        public String toString() {
+            if (value.equals(Long.MIN_VALUE)) {
+                return "BIGINT_NULL";
```

Comment:
I think that here, a brief comment would be useful to explain what this does. Does StoneDB not support Long.MIN_VALUE?

Reply:
Yes, accroding to https://stonedb.io/docs/SQL-reference/data-types

> For INT type: -2147483648 is reserved to indicate INT_NULL in Tianmu engine, Minimum Value Signed start from -2147483647.

Reply:
will add this to the comment in the code